vars/runATH.groovy

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env groovy
+
+/**
+ * Wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
+    def String metadataFile = params.get('metadataFile', 'essentials.yml')
+    def String jenkins = params.get('jenkins', 'latest')
+
+    def mirror = "http://mirrors.jenkins.io/"
+    def defaultCategory = "org.jenkinsci.test.acceptance.junit.SmokeTest"
```

Comment:
You are being inconsistent in your style here, you are using concrete objects earlier but Object here.

Reply:
(+1)